Orange/statistics/util.py

```diff
@@ -281,14 +281,32 @@ def mean(x):
     return np.sum(x.data) / n_values
 
 
-def nanmean(x):
+def nanmean(x, axis=None):
```

Comment:
Well, I did some speed testing. The results are interesting and are listed below:

> Ratio for axis  0 :  1.558
> Ratio for axis  1 :  0.664